src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import MenuIcon from '@material-ui/icons/Menu';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { AppBar, createStyles, Theme } from '@material-ui/core';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { NavBarButton, NavBarProps } from './types';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
```

Comment:
I cannot find where this class is being used, is this necessary?

Reply:
Good point: Fixed here 6968d4e7e8338a839706f0f60446463ecc90d6db and df0c5f9ab825a6913ccc7a4e0e1e2f2d2dc7f045